src/daniil_efremov_ex/ex01/tidy_code.py

```diff
@@ -0,0 +1,39 @@
+__author__ = 'Daniil Efremov'
+__email__ = 'daniil.vitalevich.efremov@nmbu.no'
+
+from random import randint as rand_num
```

Comment:
`as rand_num` is redundant here. It renames the library and make it longer, and doesn't have any use of separating `random.randint` and other `randint` from another library. And `randint` is meaningful enough so `from random import randint` is enough is this line.